src/components/containers/DashboardContainer.js

```diff
@@ -13,9 +13,12 @@ class DashboardContainer extends React.Component {
     }
 
     _onRegionChange = e => {
-        this.setState({
-            activeRegion: e.target.value,
-        });
+        const region = e.target.value;
```

Comment:
I think this is safe since react should always pass the correct event and thus target but it's still weird to see no `if (!e.target) return;` 

Just musing...definitely cleaner this way anyway.

Reply:
Ah, that's a good idea. I'll add that.